english/sonic-screwdriver/sonic-screwdriver.md

```diff
@@ -1,2 +1,8 @@
 You scan the air and hear the familiar sound of you trusty sonic-screwdriver. 
-It tells you that you are, in fact, alone.
\ No newline at end of file
+It tells you that you are, in fact, alone.
+
+You return to the Tardis, flip some levers and hear that familiar whirling sound.  When you open the door back in Londo you realize that you left your trusty sidekick behind. 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.